purchase_discount/models/purchase_order.py

```diff
@@ -110,8 +110,12 @@ def _apply_value_from_seller(self, seller):
         self.discount = seller.discount
 
     def _prepare_account_move_line(self, move=False):
-        vals = super(PurchaseOrderLine, self)._prepare_account_move_line(move)
-        vals["discount"] = self.discount
+        vals = super()._prepare_account_move_line(move)
+        if self.env["account.move.line"]._fields.get("discount1", False):
+            # OCA/account_invoice_triple_discount is installed
```

Comment:
It's a dependency of this module, so it will be anyway.

IMO it would make sense to keep your version so that in case someone only updates `purchase-workflow` it won't break. But I would add a WARNING logger in case `discount1` doesn't exists.

Reply:
> It's a dependency of this module, so it will be anyway.

No ! 

``purchase_discount`` doesn't depend on ``account_invoice_triple_discount``

See : https://github.com/OCA/purchase-workflow/blob/16.0/purchase_discount/__manifest__.py#L16

the point is that in Odoo CE : 
- purchase.order.line doesn't contain discount field, so OCA/purchase_discount implement it
- account.move.line does contain discount field, so OCA/account_invoice_triple_discount implement discount2 and discount3